### ISSUE TITLE
Add RidgeClassifier

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#64]
Add RidgeClassifier model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/classifier_config.py

Reference RidgeClassifier Implementation:
https://github.com/blobcity/ai-seed/blob/main/Classification/Linear%20Models/RidgeClassifier.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.RidgeClassifier.html

Dependencies if any, must be appropriately added. Test run of the train function on a classification problem must pass, and the function must attempt to train an RidgeClassifier as a potential best fit model.
